scripts/sbom_scraper.sh

```diff
@@ -55,37 +74,44 @@ URL=https://app.rkvst.io
 usage() {
     cat >&2 <<EOF
 
-Scrape an SBOM from a docker image and upload to abom archivist
+Create a Cyclone DX 1.2 XML SBOM from a docker image and upload to RKVST SBOM Hub
 
-Usage: $SCRIPTNAME [-p] [-c clientsecretfile] [-o output format] [-s sbomFile ] [-u url] client_id [docker-image|sbom file]
+Usage: $SCRIPTNAME [-a AUTHOR_NAME] [-A AUTHOR_NAME] [-c CLIENT_SECRET_FILE] [-e AUTHOR_EMAIL] [-s] [-p] [-u URL] CLIENT_ID [docker-image:tag|sbom file]
```

Comment:
AUTHOR_NAME twice? COMPONENT_AUTHOR_NAME perhaps for -A

Reply:
updated, thanks